packages/firebase_snippets_app/database.rules.json

```diff
@@ -1,6 +1,6 @@
 {
   "rules": {
-    ".read": "auth != null",
-    ".write": "auth != null"
+    ".read": "true",
```

Comment:
Ditto as above: if there are going to be visible to anyone outside of this repo, we should have stronger rules.